### PR TITLE
Notebooks improvements

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

- Add information about who we support in notebooks

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
@andrea-manzi should we list the specific VREs? are we missing something?

**Comment by andrea-manzi** [comment]
> @andrea-manzi should we list the specific VREs? are we missing something?

yes i think we can add the names of the projects using the notebooks in D4Science, ( SoBigData++, BlueCloud, EOSC-Pillar)

**Comment by gwarf** [comment]
@enolfc is something pending on this one, should it stay as draft?

File: content/en/users/notebooks/_index.md [hunk 1]
```diff
@@ -72,3 +72,13 @@ Organisation (i.e. federated compute and storage sites) for Jupyter, add custom
 libraries into Jupyter (e.g. discipline-specific analysis libraries) or have
 fine grained control on who can access the instance (based on the information
 available to the EGI Check-in AAI service).
+
+EGI currently operates community instances for:
+
+- [D4Science](https://www.d4science.org/). These instances are accessed through
+  specific VREs: [SoBigData](https://sobigdata.d4science.org/),
```
Comment (andrea-manzi) [inline content/en/users/notebooks/_index.md:79]:
```suggestion
  specific Gateways: [SoBigData](https://sobigdata.d4science.org/),
```

File: content/en/users/notebooks/_index.md [hunk 2]
```diff
@@ -72,3 +72,13 @@ Organisation (i.e. federated compute and storage sites) for Jupyter, add custom
...
+  [Blue-Cloud](https://blue-cloud.d4science.org/) and
```
Comment (andrea-manzi) [inline content/en/users/notebooks/_index.md:80]:
```suggestion
  [Blue-Cloud](https://blue-cloud.d4science.org/), [D4Science Services](https://services.d4science.org/) and 
```

File: content/en/users/notebooks/_index.md [hunk 3]
```diff
@@ -72,3 +72,13 @@ Organisation (i.e. federated compute and storage sites) for Jupyter, add custom
...
+  [EOSC-Pillar](https://eosc-pillar.d4science.org/). Check with your VRE manager
```
Comment (andrea-manzi) [inline content/en/users/notebooks/_index.md:81]:
```suggestion
  [EOSC-Pillar](https://eosc-pillar.d4science.org/). Check with the [D4Science support](https://www.d4science.org/contact-us) 
```

**Comment by andrea-manzi** [comment]
@enolfc I did small changes

**Review by andrea-manzi — APPROVED** [review]
LGTM

**Comment by gwarf** [comment]
The user documentation got reorganised via #414, can you please resolve the conflict in your PR?

**Comment by enolfc** [comment]
> The user documentation got reorganised via #414, can you please resolve the conflict in your PR?

Done

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**